### PR TITLE
Chapter 1 Solution

### PR DESCRIPTION
**Pull request by khvmaths**
### Solutions for Chapter 1

Thank you for the great tutorial! Lovin' them so far 👍 

_ps: i deleted the last pr because i didnt checkout them seperately_

cc @vrom911 @chshersh

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -619,8 +624,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit + secLastDigit
+  where
+    lastDigit = mod n 10
```
Comment (vrom911) [inline src/Chapter1.hs:630]: You can notice that this is the same as `lastDigit` function from the previous task 🙂 

Also, here, you also need to consider the case with negative numbers 😉

File: src/Chapter1.hs [hunk 2]
```diff
@@ -640,8 +648,12 @@ Implement a function that returns the first digit of a given number.
...
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n =
+  let m = abs n
+  in if m < 10
+    then m
+    else firstDigit(div m 10)
```
Comment (vrom911) [inline src/Chapter1.hs:656]: Great one! 🌟

**Review by vrom911 — APPROVED** [review]
Fantastic job on the Chapter 🎉 
And great work with the advanced tasks 🤩

**Comment by khvmaths** [comment]
Thanks @vrom911 for reviewing 💯

**Comment by vrom911** [comment]
The solutions look good :+1: I'm going to close this PR, because we can't merge the solutions to our repo :slightly_smiling_face: But don't worry, it still counts towards Hacktoberfest contribution :shirt: :deciduous_tree: Feel free to open PRs with solutions to other chapters!

**Pull request closed without merging**